omod/src/test/java/org/openmrs/module/aijar/page/controller/MarkPatientDeadTest.java

```diff
@@ -0,0 +1,95 @@
+package org.openmrs.module.aijar.page.controller;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Concept;
+import org.openmrs.Patient;
+import org.openmrs.api.context.Context;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+
+import java.util.Date;
+
+/**
+ * Created by lubwamasamuel on 19/15/2016.
+ */
+public class MarkPatientDeadTest extends BaseModuleWebContextSensitiveTest {
```

Comment:
@slubwama I was responding to your comment. On the https://github.com/METS-Programme/openmrs-module-aijar/pull/169/commits/39db94d5b3d8bf5f75869aa24db38bfdcbb2d631 see the comment that is still open